src/java.base/unix/classes/sun/security/provider/NativePRNG.java

```diff
@@ -508,8 +508,8 @@ private static void readFully(InputStream in, byte[] data, Variant variant)
                 if (in.readNBytes(data, 0, len) < len) {
                     throw new IOException("Could not read from file(s)");
                 }
-            }
-            /*[ELSE] CRIU_SUPPORT */
+            } else {
+            /*[ENDIF] CRIU_SUPPORT */
```

Comment:
Suggest using `return;` after line 510 eliminating the need for `else` and lines 525-527 below.